docs/DeveloperGuide.md

```diff
@@ -209,7 +253,28 @@ The second sequence diagram given below shows the detailed interaction that acce
 
 ![CalcReffSequence](diagrams/CalcReffSequenceDiagram.png)
 
-<small><i>Figure 14</i></small>  
```

Comment:
This shouldn't be removed right?

Reply:
Strange, it seems like my fork still has this even though its been updated